dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -755,6 +763,35 @@ void CommandBarFlyoutCommandBar::EnsureFocusedPrimaryCommand()
     }
 }
 
+void CommandBarFlyoutCommandBar::MoreButtonClicked(const winrt::IInspectable& sender, const winrt::RoutedEventArgs& args)
+{
+    if (m_secondaryCommandsExpectedToBeVisible)
```

Comment:
Why is this needed here?

Reply:
Forgot to remove that, that was just debug code. Thanks for pointing that out!